salt/states/pkgbuild.py

```diff
@@ -237,7 +237,7 @@ def built(name,
     return ret
 
 
-def repo(name, keyid=None, env=None):
+def repo(name, keyid=None, env=None, use_passphrase=False, gnupghome='/etc/salt/gpgkeys', runas='builder'):
```

Comment:
that is the default user for salt-pack which is used in conjunction with these functions, on Redhat